src/esp/assets/ResourceManager.cpp

```diff
@@ -1206,13 +1358,14 @@ void ResourceManager::addPrimitiveToDrawables(int primitiveID,
                                               DrawableGroup* drawables) {
   CHECK(primitiveID >= 0 && primitiveID < primitive_meshes_.size());
   createDrawable(ShaderType::COLORED_SHADER, primitive_meshes_[primitiveID],
-                 node, drawables);
+                 node, Corrade::Containers::NullOpt, drawables);
 }
 
 void ResourceManager::createDrawable(
     const ShaderType shaderType,
     Magnum::GL::Mesh& mesh,
     scene::SceneNode& node,
+    Corrade::Containers::Optional<uint32_t> meshID, /* = NullOpt */
```

Comment:
We seem to be over-loading createDrawable with a lot o responsiblities. Not sure if the code might be simpler if it the new functionality were added to addMeshToDrawables instead.